DQM/HcalCommon/interface/ContainerSingle2D.h

```diff
@@ -0,0 +1,95 @@
+#ifndef ContainerSingle2D_h
+#define ContainerSingle2D_h
+
+/*
+ *	file:			ContainerSignle2D.h
+ *	Author:			Viktor Khristenko
+ *
+ *	Description:
+ *		Container to hold a single ME - for convenience of initialization
+ */
+
+#include "DQM/HcalCommon/interface/Container.h"
+#include "DQM/HcalCommon/interface/ValueAxis.h"
+#include "DQM/HcalCommon/interface/CoordinateAxis.h"
+
+#include <string>
+
+namespace hcaldqm
+{
+	using namespace axis;
+	class ContainerSingle2D : public Container
+	{
+		public:
+			ContainerSingle2D();
+			ContainerSingle2D(std::string const& folder, 
+				std::string const& nametitle, 
+				axis::Axis *xaxis,
+				axis::Axis *yaxis = new CoordinateAxis(fYaxis, axis::fiphi), 
+				axis::Axis *zaxis = new ValueAxis(fZaxis, fEntries));
+			virtual ~ContainerSingle2D() {}
```

Comment:
@vkhristenko - all of these Container classes would seem to leak the axis pointers. Please delete them